src/attached_probe.h

```diff
@@ -84,4 +84,16 @@ class AttachedProbe
   std::function<void()> usdt_destructor_;
 };
 
+class HelperVerifierError
```

Comment:
Should inherit from `std::runtime_error` here. So catching by `std::exception &` will work